src/asli/asli.py

```diff
@@ -490,6 +497,20 @@ def _get_cli_calc_args():
         default=1,
         help="Number of processes used by joblib in parallel calculation.",
     )
+    parser.add_argument(
+        "-M",
+        "--minima",
+        type=int,
+        nargs="?",
+        default=3,
+        help="Max number of minima to locate in pressure field per time step."
```

Comment:
Three minima are only in fact returned when the `-a` argument is also present. Is that the intended behaviour? What is the significance of having a default of 3 when 1 is always returned without `-a`?

It might be useful to clarify the relationship between `-M` and `-a`.

Reply:
```suggestion
        help="Max number of minima within the peak intensity threshold to locate in pressure field per time step."
```

Adding a few words on the threshold, to clarify instances where n peaks < `-M` are returned.

Reply:
I've actually removed the threshold in this PR (1fb0f4ab4cabbe3732f7aa27c80e099498b435b4) as it doesn't make any difference to the number of peaks found.

Reply:
> Three minima are only in fact returned when the `-a` argument is also present. Is that the intended behaviour? What is the significance of having a default of 3 when 1 is always returned without `-a`?

Good point, this kind of goes back to the slightly odd way this was written in the first place. The number of minima arg controls how many `peak_local_max` is told to look for, then only the top one is returned unless `-a` is on. So maybe we don't need `-a`? Just use `-M` on its own?

Reply:
Done [9833df9](https://github.com/davidwilby/amundsen-sea-low-index/pull/37/commits/9833df91ab8bea6b27a0b98068d10d0af57c7d12)

Reply:
Sweet!